crates/driver/src/infra/config/mod.rs

```diff
@@ -12,7 +12,14 @@ pub struct Config {
     pub disable_gas_simulation: Option<eth::Gas>,
     pub solvers: Vec<solver::Config>,
     pub liquidity: liquidity::Config,
-    pub tenderly: Option<simulator::tenderly::Config>,
+    pub simulator: Option<Simulator>,
     pub mempools: Vec<mempool::Config>,
     pub contracts: blockchain::contracts::Addresses,
 }
+
+/// Configuration of the transaction simulator.
+#[derive(Debug)]
+pub enum Simulator {
+    Tenderly(simulator::tenderly::Config),
+    Enso(simulator::enso::Config),
+}
```

Comment:
nit: this should be probably moved somewhere else, for example `crates/driver/src/infra/simulator/mod.rs` or `crates/driver/src/infra/simulator/config.rs` to keep this file clean and be consistent.